crates/fluvio-smartmodule/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-smartmodule"
-version = "0.7.3"
+version = "0.8.0"
```

Comment:
maybe 0.7.4 instead of 0.8

Reply:
Or is there any reason to prefer 0.8 instead?

Reply:
Changed to 0.7.4. 

Maybe because it's a breaking change?

Reply:
Not sure if this change on fluvio-protocol will be an issue for next releases

Reply:
It will be if we maintain the current compatibility handshake, but we will just have to keep in it sync for sdf too until we can improve that.